pennylane_qiskit/vqe_runtime_runner.py

```diff
@@ -0,0 +1,499 @@
+
+    Returns:
+        str: Program id that can be used to run the program.
+    """
+
+    connect(kwargs)
+
+    # Specify a single hub, group and project
+    hub = kwargs.get("hub", "ibm-q")
+    group = kwargs.get("group", "open")
+    project = kwargs.get("project", "main")
+
+    meta = {
+        "name": "vqe-runtime",
+        "description": "A sample VQE program.",
+        "max_execution_time": 100000,
+        "spec": {},
+    }
+
+    meta["spec"]["parameters"] = {
+        "$schema": "https://json-schema.org/draft/2019-09/schema",
+        "properties": {
+            "hamiltonian": {
+                "description": "Hamiltonian whose ground state we want to find.",
+                "type": "array",
+            },
+            "x0": {
+                "description": "Initial vector of parameters for the quantum circuit.",
+                "type": "array",
+            },
+            "ansatz": {
+                "description": "Qiskit circuit or name of ansatz quantum circuit to use, default='EfficientSU2'",
+                "type": "[QuantumCircuit,string]",
+                "default": "EfficientSU2",
+            },
+            "ansatz_config": {
+                "description": "Configuration parameters for the ansatz circuit.",
+                "type": "dict",
+            },
+            "optimizer": {
+                "description": "Classical optimizer to use, default='SPSA'.",
+                "type": "string",
+                "default": "SPSA",
+            },
+            "optimizer_config": {
+                "description": "Configuration parameters for the optimizer.",
+                "type": "dict",
+            },
+            "shots": {
+                "description": "The number of shots used for each circuit evaluation.",
+                "type": "integer",
+            },
+            "use_measurement_mitigation": {
+                "description": "Use measurement mitigation, default=False.",
+                "type": "boolean",
+                "default": False,
+            },
+        },
+        "required": ["hamiltonian", "x0"],
+    }
+
+    meta["spec"]["return_values"] = {
+        "$schema": "https://json-schema.org/draft/2019-09/schema",
+        "description": "Final result in SciPy optimizer format",
+        "type": "object",
+    }
+
+    meta["spec"]["intermadiate_results"] = {
+        "$schema": "https://json-schema.org/draft/2019-09/schema",
+        "description": "Dictionnary containing: "
+        "The number of evaluation at current optimization step."
+        "Parameter vector at current optimization step."
+        "Function value at the current optimization step."
+        "The size of the step.",
+        "type": "dict",
+    }
+
+    provider = IBMQ.get_provider(hub=hub, group=group, project=project)
+    ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    program_path = os.path.join(ROOT_DIR, "runtime_programs/vqe_runtime_program.py")
```

Comment:
LIkely `"runtime_programs/vqe_runtime_program.py"` will also have to be split and joined using `os.path.join` for compatibility on Windows